signinapp/model.py

```diff
@@ -291,6 +303,33 @@ def get_from(cls, name: str, phone_number: str, email: str, contact_order: int)
         return guardian_user_data
 
 
+class Student(db.Model):
+    __tablename__ = "students"
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"))
+
+    # Extra student information
+    graduation_year = db.Column(db.Integer)
+
+    # One to One: Links User to extra student information
+    user = db.relationship("User", back_populates="student_user_data")
+
+    @property
+    def display_grade(self):
+        grades = generate_grade_choices()
+        return grades[self.graduation_year]
```

Comment:
Should we have a check here to prevent invalid lookups (i.e. looking at an old database with a student who has graduated)

Reply:
Good point. I'll add a default for "Alumni"